src/reducers/filter.js

```diff
@@ -0,0 +1,8 @@
+export const filter = (state = '', action) => {
```

Comment:
let this Default to `All` as stated in project requirements. `state='All`

Reply:
@SunnyAsar
Thanks for the review.
But in the project requirements, we were asked to avoid storing 'All' in the Redux store (2nd requirement, 2nd point)